### ISSUE TITLE
Validation of connections between monitored roads.

### ISSUE THREAD

**Issue by kendzi** [ppawel/osmonitor#14]
Wikipedia pages contains information about direction of way and many connected ways. Eg. 

http://pl.wikipedia.org/wiki/Droga_wojew%C3%B3dzka_nr_303 

Shows that dw303 is near / connected with roads 92, 302, 304, 313 and 32

So it is easy to extract this information form wiki. Monitor can check connection between road and data in wiki page.

From wiki it can look like table with columns
Road Id | List of connected roads | notes | status
dw303   | 302, 304...                        | …      | raport

Connection between roads can be easy check (union of nodes) and maybe check of short link connection.

**Comment by ppawel** [comment]
Yes that's possible but we talked on IRC yesterday about moving all input information from the wiki to relations - it is just easier to maintain it. On the wiki it sucks to put too much stuff in one column. So relation for a road would have tags like:

qa:road_length=50

Info about connected roads also could be in relation tags.
